test/hooks/useContactsList.test.jsx

```diff
@@ -53,7 +53,7 @@ describe('useContactsList', () => {
     givenName: 'Zoro',
     familyName: 'Roronoa',
     webId: 'http://www.example.com/swords',
-    person: 'Zoro Roronoa',
+    person: null,
```

Comment:
Given that `person` would no longer be in use, rather than changing this to `null`, the line itself can be removed entirely.

Reply:
Also, given the fact that you've updated the function that serializes the Solid object/Thing, the function that makes the mock Thing from Line 42 to Line 49 needs to be updated to match the new changes (see the changes you've done for the `serialize` function in `src/hooks/useContactsList.js` hook and replace the function inside `makeIntoThing` with that new function). You could keep the name as `makeIntoThing` if you'd like

Reply:
okay, i've made another request addressing all of these. I apologize if it has been sloppily done. Thanks for your time kahung

Reply:
No worries Fig, it's what reviews are for, to weed out things someone might have missed. I, for one, think you're doing just fine.